packages/react/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -136,6 +136,36 @@ exports[`styled Renders basic component 1`] = `
 }
 `;
 
+exports[`styled Renders components from other libraries 1`] = `
+{
+  "orderedAppliedStyles": [
+    "._vfarC {color: red;}",
+    "._vfarC {color: red;}"
```

Comment:
This looks a bit off. It's rendering it twice. Is this normal? @hadihallak

Reply:
good catch 💯 
I'm hoping this is a bug with the jest serializer but if not then there might be an issue with the atom caching functionality